assets/css/main.css

```diff
@@ -243,3 +243,19 @@ img {
   background-image: linear-gradient(45deg, rgba(255, 255, 255, 0.15) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.15) 50%, rgba(255, 255, 255, 0.15) 75%, transparent 75%, transparent);
   background-size: 10px 10px;
 }
+
+body.layout-editing .droppable {
+  min-height: 50px;
+}
+
+body.layout-editing .droppable:nth-of-type(3n) {
+  background-color: rgba(0, 0, 0, 0.25);
```

Comment:
would prefer it didn't add any extra coloring, personally

Reply:
It's easier to distinguish columns this way.

Reply:
it also looks janky in the preview you have up